src/olympia/reviewers/templates/reviewers/includes/version.html

```diff
@@ -60,7 +60,16 @@
       {% endif %}
 
       {% if version.needs_human_review %}
-        <div><strong class="risk-high">{{ _('Flagged by scanners') }}</strong></div>
+        <div><strong class="risk-high">
+          {{ _('Flagged by scanners') }}
+          {%- if request.user.is_staff and action_allowed(amo.permissions.ADMIN_SCANNERS_RESULTS_VIEW) %}
```

Comment:
Why do we need the `is_staff` restriction?

Reply:
Requested in the issue. It's unlikely to happen in production, but locally or on dev, users might have the permission while not having access to the admin in general because they don't have `is_staff`. The extra condition ensures those users wouldn't see a broken link.